modules/hitlnext/src/config.ts

```diff
@@ -16,6 +16,12 @@ export interface Config {
    */
   messageCount: number
 
+  /**
+   * @param handoffAlert Number of messages to display in the conversation history
```

Comment:
I'm pretty sure the description is off ;)

Reply:
changed to: Amount of time in minutes before the handoff is turning yellow.